mod/src/main/kotlin/moe/nea/ledger/modules/ForgeDetection.kt

```diff
@@ -21,6 +21,7 @@ class ForgeDetection {
 		val slot = event.slotIn ?: return
 		val clickedItem = slot.stack ?: return
 		if (clickedItem.displayName.unformattedString() != "Confirm") return
+		if (clickedItem.itemDamage == 14) return
```

Comment:
Can you use `EnumDyeColor.byDyeDamage` (or the corresponding method for the item type here)